controllers/conditions.go

```diff
@@ -42,3 +42,61 @@ func NotReadyCondition(reason, message string) sourcev1.SourceCondition {
 		Message:            message,
 	}
 }
+
+func ReadyGitRepository(repository sourcev1.GitRepository, artifact sourcev1.Artifact, url, reason, message string) sourcev1.GitRepository {
```

Comment:
I would move these functions to their `<kind>_types.go` file or a new `<kind>_conditions.go` file given they are kind specific.

Reply:
I moved them to types.